### PR TITLE
Add parsefolder param and logic to support it

### PR DESCRIPTION
**Pull request by asmorodskyi**

### CODE REVIEW

File: export.py [hunk 1]
```diff
@@ -4,28 +4,43 @@
...
+            response = requests.post(url, data=request_string)
+            logging.info('Response from DB: {0}'.format(response.content))
 
 logging.basicConfig(filename='export.log', level=logging.INFO,
                     format='%(asctime)s - %(name)s - %(levelname)s - %(message)s')
 parser = argparse.ArgumentParser()
-parser.add_argument('--parsefile', required=True)
+parser.add_argument('--parsefile')
+parser.add_argument('--parsefolder', default='/tmp')
 parser.add_argument('--targeturl', default='http://fromm.arch.suse.de:8086')
```
Comment (jlausuch) [inline export.py:31]: we should add environment information as parameters:
- os_version
- os_build
- vswitch_version

File: export.py [hunk 2]
```diff
@@ -4,28 +4,43 @@
...
+import re
+import os
+
+def parse_csv(parsefile):
+    with open(parsefile, 'rb') as csvfile:
+        csvreader = csv.DictReader(csvfile, delimiter=',')
+        for row in csvreader:
+            request_string = 'throughput,testcase={0},traffic_type={1},' \
+                .format(row['type'], row['traffic_type'])
+            request_string += 'packetsize={0},vswitch={1},hw={2} value={3}' \
```
Comment (jlausuch) [inline export.py]: as discussed, I think we don't need the HW description here, as it will be always the same

File: export.py (hunk 1 above)
Comment (jlausuch) [reply export.py:31]: and what about a link to the openqa run?

Comment (asmorodskyi) [reply export.py:31]: done

**Comment by jlausuch** [comment]
LGTM

**Pull request merged**